go/client/cmd_chat_api.go

```diff
@@ -311,7 +311,6 @@ func (c *CmdChatAPI) ReadV1(ctx context.Context, opts readOptionsV1) Reply {
 			return c.errReply(libkb.NewChatMessageVersionError(version))
 		}
 
-		return c.errReply(errors.New("unexpected response from service: UnboxingError and Message are both empty"))
```

Comment:
It looks like this should be there, but the issue is we are missing a `continue` after line 309?